### DISCUSSION TITLE
Impossible to write "wget" followed by an url

### DISCUSSION THREAD

**Discussion by bob-png-debug** [requarks/wiki#6343] [Error / Bug Report]
In the markdown editor, when writing `wget https://whatever` or `curl https://whatever`, it becomes impossible to save the article.

I do updates regularly and some old articles who contain those commands are not editable anymore (You can edit them but it won't save). So it must be a recent issue.

Really easy to reproduce : 
- Create an article using mardown editor
- Type "wget http://" or "curl https://"
- Hit save
- "Network error: Failed to fetch"

**In short : wget or curl + url => Network error: Failed to fetch**

_I am running Wiki.js 2.5.298_

**Comment by NGPixel** [comment]
You are most likely using Cloudflare or some other reverse proxy? You need to allowlist your IP or disable the rules that block the save request.

**Reply by bob-png-debug** [reply]
Thank you for your message! Just to clarify, we're using traefik as our reverse proxy, and there is no rule currently blocking the save request. I've been able to save other articles without any issues. 

Even when setting the log level to debug, I see no logs on the wiki side.

Please let me know if there's any other details I should provide.
